packages/eslint-config-udemy-website/index.js

```diff
@@ -140,6 +142,80 @@ module.exports = {
                 },
             ],
         ],
+        'udemy/udlite-import-blacklist': [
+            'error',
+            [
+                {
+                    source: 'base-components/(?!udlite/)',
+                    message:
+                        'UDLite files may not import UDHeavy base-components. ' +
+                        'Replace the following with a base-components/udlite/ component:',
+                },
+                {
+                    source: '(react-bootstrap|react-overlays|react-router-bootstrap)',
+                    message:
+                        'UDLite files may not import React-Bootstrap libs. ' +
+                        'Replace the following with a base-components/udlite/ component:',
+                },
+                {
+                    source: '(react-popper|popper.js)',
+                    message:
+                        'UDLite files may not import Popper. ' +
+                        'Replace the following with a base-components/udlite/ component:',
```

Comment:
@cwick do you know what the plan is for popovers in UDLite? I remember we don't want them in UDLite mobile from a design perspective, but for UDLite desktop we might use a popover to implement dropdown menus in the header? In any case, if we do end up having a UDLite popover, I hope it doesn't have to use this Popper lib. The UDHeavy popover is very complex.

Reply:
Guess that means we're implementing our own lite version of a popover.